package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dpl",
-  "version": "4.0.0",
+  "version": "4.0.1",
```

Comment:
this is an auto-generated file. not much we can do to "review" it. 🙄 
A `mix.lock` file in `Elixir` is _so_ much better!
see: https://github.com/dwyl/email/blob/master/mix.lock
All dependencies are _flat_ (_not infinitely nested_) and transparent. 😍